tests/views_integration/Tribe/Events/Views/V2/ExtendingViewTest.php

```diff
@@ -111,14 +114,23 @@ public function should_fetch_the_template_from_the_theme_if_available() {
 	/**
 	 * @test
 	 */
-	public function should_render_the_not_found_view_if_trying_to_render_not_registered_view() {
-		$this->markTestSkipped( 'Skipping the 404 since we need to revist the behavior here.' );
-
-		$view = View::make( 'not-set' );
+	public function should_render_the_default_view_if_view_not_found() {
```

Comment:
Does this reflect a change in behavior?

Reply:
Yes, we used to print a "not found" view template, we now print the default view template.